hal/halx86/apic/halinit.c

```diff
@@ -63,7 +63,7 @@ HalpInitPhase0(IN PLOADER_PARAMETER_BLOCK LoaderBlock)
     HalpEnableInterruptHandler(IDT_DEVICE,
                                0,
                                APIC_PROFILE_VECTOR,
-                               PROFILE_LEVEL,
+                               APIC_PROFILE_LEVEL,
```

Comment:
I guess this is what we would want,
but can code outside APIC HAL cope with this change?
https://jira.reactos.org/browse/CORE-20093?focusedId=144726&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-144726